roles/rsyslog/vars/inputs/ovirt/main.yml

```diff
@@ -95,7 +105,7 @@ rsyslog_conf_ovirt_formatting:
               set $.send_targets_only = "true";
           }
 
-          {% if collect_ovirt_engine_log %}
+          {% if collect_ovirt_engine_log | d(true) | bool %}
```

Comment:
here `collect_ovirt_engine_log` appears to be a global variable, not the `logging_inputs[0].collect_ovirt_engine_log` set below?